client/app/sidenav/employee/employee.html

```diff
@@ -33,10 +35,11 @@ <h4 layout="row" layout-align="start" style="padding-right: 5px">Ksquare Employe
 					<span style="fill: white;" flex></span>
 
 
-					<select ng-show="vm.showStatus">
-						<option value="activate">Activate</option>
-						<option value="deactivate">Deactivate</option>
-						<select>
+					<md-input-container>
```

Comment:
right now when you select one or more employees it will keep the select box on the option that was last chosen. then if you change your choice, and select another one it doesn't change to the current selected. a work around would be to have a drop down menu that has the title of "active status" then you have the choices of activate / deactive. It still will work the same but since the active column has to be scrolled over to see, it may make it a little confusing to people that don't normally use this app.